pages/checkout/_add-ons.html

```diff
@@ -41,6 +41,10 @@ <h1 class="addon-amount" id="addon-amount-refarch">$4,950</h1>
             <h3 class="addon-type">CIS Benchmark Compliance</h3>
             <hr class="hr-sm hr-reset visible-sm-block visible-md-block visible-lg-block">
             <p>Achieve and maintain compliance with the CIS AWS Foundations Benchmark.</p>
+            <p class="text-muted small text-center margin-bottom-none visible-sm-block visible-md-block visible-lg-block">
+              Add <a href="#reference-architecture">Reference Architecture</a> to get a CIS compliant end to end
+              production-grade AWS architecture in 1 day.
```

Comment:
See above

Reply:
Updated abdb435 

But not sure if it looks good with the longer text

<img width="541" alt="Customize_your_plan_with_add-ons" src="https://user-images.githubusercontent.com/430092/71299475-4a532c80-2342-11ea-9a85-d4a4a37b2682.png">

Thoughts?

Reply:
I'd consider phrasing it as follows:

> Achieve and maintain compliance with the CIS AWS Foundations Benchmark. Our code is certified compliant by the Center for Internet Security. Add the Reference Architecture to get an end-to-end, production-grade architecture in your own AWS environment, fully managed as code - all in about 1 day!

I'd also leave it as one paragraph with the same font.

Reply:
Done! 3c2e40d 

Making it one paragraph does look better:

<img width="546" alt="Customize_your_plan_with_add-ons" src="https://user-images.githubusercontent.com/430092/71300892-38778680-234e-11ea-930f-44767104f97a.png">